src/main/java/org/mitre/synthea/world/agents/behaviors/planeligibility/CSVEligibility.java

```diff
@@ -0,0 +1,118 @@
+package org.mitre.synthea.world.agents.behaviors.planeligibility;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.mitre.synthea.world.agents.Person;
+
+public class CSVEligibility implements IPlanEligibility {
+
+  // The possible columns of the CSV input file.
```

Comment:
I think there is a little bit of a user interface issue to discuss here.

The users of CSV Eligibility files are going to be mostly non-developers. However, column names are in snake case. This isn't a big deal, but it is worth considering if these should be in just normal title case or if the code should be a bit more robust to things like capitalizing parts or all of the column names.

Reply:
I've updated the csv eligibility columns to use white spaces and capitalizations... As a larger approach, though, should this same format be extended to Payers and Plans?